src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -401,10 +401,10 @@ private function configureAdapters(ContainerBuilder $container, array $config):
     {
         foreach (['gd', 'imagick', 'gmagick'] as $adapter) {
             if ($container->hasParameter('sonata.media.adapter.image.'.$adapter.'.class')) {
-                $container->register(
-                    'sonata.media.adapter.image.'.$adapter,
-                    $container->getParameter('sonata.media.adapter.image.'.$adapter.'.class')
-                );
+                $adapterClass = $container->getParameter('sonata.media.adapter.image.'.$adapter.'.class');
+                \assert(\is_string($adapterClass));
```

Comment:
better to cast this as string or assert is_string?

Reply:
In SonataAdmin I used assert, since IMHO it's always a string according to the config.